cdc/processor/sinkmanager/manager.go

```diff
@@ -151,7 +151,8 @@ func New(
 		m.redoWorkers = make([]*redoWorker, 0, redoWorkerNum)
 		m.redoTaskChan = make(chan *redoTask)
 		m.redoWorkerAvailable = make(chan struct{}, 1)
-		// TODO: maybe should use at most 1/3 memory quota for redo event cache.
+		// Use 3/4 memory quota as redo event cache.
+		m.eventCache = newRedoEventCache(changefeedID, changefeedInfo.Config.MemoryQuota/4*3)
```

Comment:
Is  3/4 more reasonable than 1/3?

Reply:
I found that 1/3 or 1/2 can lead low cache hit ratio (about 15%~20%) on heavy load. I guess 3/4 is helpful. I will examine it when I do performance tests.

Reply:
Could you please add these comments to the code?